_caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md

```diff
@@ -0,0 +1,140 @@
+
+The conducting of a needs assessment that the author had thought would only take a day, had taken an entire week. The week focused on getting oriented to the teaching staff’s ways of engagement and eliciting their requirements for the e-learning course, by employing various real-time evaluation techniques such as impromptu interviews, focus groups and constant observations. The observations from each day informed the course of the next, and the evaluation continued until there was sufficient information to design the required learning materials. Patricia Rogers (Better Evaluation, 2020) aptly describes real-time evaluation as a combination of the following factors: real-time data, real-time reporting, multiple cycles, double loop learning and diverse sensemaking.
+
+![](/assets/img/posts/RealTimeEvaluations_PRogers_v2.png)
+
+The needs assessment process can consist of mixed iterations of the following techniques: describing current reality vs. desired future expectations, engaging stakeholders relevant to the intended outcome, observing people and/or systems at work, identifying root causes and defining the problem, collecting self-assessments & peer reviews, and gathering evidence through questionnaires.
+
+![](/assets/img/posts/NeedsAssessmentTechniques.png)
+
+Many of these steps were employed in various ways and sequences in order to determine the objectives, content, and structure of the e-learning methods course to be given to the teaching staff for the remainder of the time spent at URDT.
+
+### How we incorporated HCD
+
+The [MERL Center definition of HCD](https://merlcenter.org/human-centered-design-definition.html) is as follows:
+> “Human-centered design (HCD) is a framework that places the needs, desires and behaviors of key stakeholders, beneficiaries, users or teaching staff at the center of design and implementation decisions. HCD can be used to create and inform digital products, physical products, programs and communities. It can rely on iterative cycles of co-creating, collaborating, testing and refining solutions. HCD requires contextual analysis and understanding in order to base design decisions on how stakeholders or end-users already think, communicate and engage. In MERL, HCD can be used to inform who is asked what, when and how. It can provide insights on why something is happening. In using HCD for MERL, it is important to consider human rights and equitable solutions.”
+
+
+In an attempt to align with this definition, it is necessary to note that, not only was it important to figure out what the teaching staff needed in order to continue teaching when students are not physically at school, it was also important to figure out how to elicit accurate and relevant information for an effective course delivery.
```

Comment:
@mutsacc 
Please see screenshot. There needs to be a space between the grey box and the next paragraph starting with ` In an attempt.`
<details>
  <summary>Screenshot</summary>
   <img src = "https://github.com/MERLTech/MERL-Center-public/assets/75643389/b98978f3-68b1-4c24-b431-c24f72b45362">

</details>